tests/python/gaia-ui-tests/gaiatest/apps/calendar/app.py

```diff
@@ -15,6 +15,7 @@
 class Calendar(Base):
 
     name = 'Calendar'
+    origin = 'app://calendar.gaiamobile.org'
```

Comment:
How would this work? It's self.apps.displayed_app.name that changes its name depening on the locale that is used.